Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:12.7.0-alpine
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+ARG GIT_TAG
+
```

Comment:
You need to pass arg to env

Reply:
@byliuyang 
Is it a convention to pass ARG to ENV?
Otherwise, having it as ARG works. [Build Test](https://ci.time4hacks.com/rohithbalaji123/short-ext/16/1/2), `testArg` is the tag name. You can see it being resolved in the manifest.json file which is printed at the end of the file.

Reply:
@rohithbalaji123 https://vsupalov.com/docker-arg-vs-env/ Let's just keep ARG because we only use `GIT_TAG` at build time.